include/ros2_logging_fmt/ros2_logging_fmt.hpp

```diff
@@ -15,49 +15,113 @@ class Logger
   Logger(rclcpp::Logger rclcpp_logger): rclcpp_logger_(rclcpp_logger)
   {}
 
+  Logger(rclcpp::Logger rclcpp_logger, rclcpp::Clock::SharedPtr clock): rclcpp_logger_(rclcpp_logger)
+  { clk = clock; }
+
   template<typename... Args>
   void info(const char* str, Args... args)
   {
     RCLCPP_INFO(rclcpp_logger_, write_buffer(str, args...) );
   }
 
+  template<typename... Args>
+  void info_throttle(const char* str, rclcpp::Clock::SharedPtr clock, int duration, Args... args)
```

Comment:
Would it not be better to take in a `rclcpp::Duration` or `chrono` type for the time instead of an integer?